src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+
+Command and Security are the only roles that have mindshields roundstart.
+
+- A mindshield gives its owner a "shield" with **100** LHP. The mindshield absorbs any LHP damage dealt to the owner and takes it instead.
+- The shield slowly regains LHP on its own, at **0.2** LHP / second. This functionality does not work when converted to the Revolution. 
+- Having a mindshield does not change Revolutionary status or antag faction; a Revolutionary can be mindshielded and still belong to the Revolution.
+- A newly injected mindshield always has full LHP, regardless of Revolutionary status.
+- Mindshields can be removed, much like any other implant. The exceptions for this is the Captain and ERT/CentComm, who have unremovable mindshields (however due to being unconvertable, this is mostly just flavor). 
+
+New mindshields are able to be purchased from Cargo at a reasonable price. One possible suggestion would be 4 implanters for $8000.
+
+## Revolutionary Items
+
+To assist in the revolution, Revolutionaries have access to a multitude of items. Some are generally available regardless of roundtype, while some are specific to the Revolutionary antag. The choice and utilization of these items should provide players options and flavors for how the revolution plays out. It's important that the items do not encourage gameplay that go against the Revolutionary Gamemode Goals.
+
+### Generally available items
+
+**Headbands**
+- Headband is a headwear item that can be created by using a sharp object on a bandana. (Crafting method may be changed if dyeing clothes is implemented).
+- This is a fashion accessory accessible to all crew, however when worn by a Revolutionary their codewords deal 1.5x more LHP damage.
+
+**Revolutionary posters**
+- Crew can print rolled posters from autolathes using cloth. This feature is available regardless of roundtype.
+- These posters are the same posters that spawn on stations roundstart. 
+- Posters are randomized when printed, and can result in loyal, neutral or revolutionary posters. Examining a rolled poster gives its name and allegiance.
+- Revolutionary posters have a **4** tile radius of influence when put up (requires line of sight).
+  - When inside this radius, you take **0.25** LHP damage / second (to a max of **4** stacks).
+  - When inside this radius, codewords deal **1.25**x more damage (does not stack with itself).
+- Loyal posters also have a **4** tile radius of influence when put up (requires line of sight).
+  - When inside this radius, you heal **0.25** LHP damage / second (to a max of **4** stacks).
+  - When inside this radius, codewords deal **0.5**x damage (does not stack with itself).
+- Posters can be taken down, and if recycled gives back some cloth.
+- Destroying a poster that is rolled up on a wall heals **5** LHP if it's a Revolutionary poster, or deals **10** LHP if it's a Loyal poster.
+
+### Dead Drop Duffelbag / The Revolutionaries Uplink 
```

Comment:
Skimming over this section but there's some great ideas in here. Especially love the megaphone.